Packages/com.unity.inputsystem/InputSystem/Actions/InputBinding.cs

```diff
@@ -841,6 +843,18 @@ public bool Matches(InputBinding binding)
         {
             return Matches(ref binding);
         }
+        
+        internal bool MatchesMask(ref InputBinding binding)
+        {
+            var appendix = path.Substring(path.Length-2,path.Length-(path.Length-2));
+            if (appendix == wildcard)
```

Comment:
I'm concerned this is very specific to /* and not the wider wild card support listed in documentation here:

https://docs.unity3d.com/Packages/com.unity.inputsystem@1.0/manual/ActionBindings.html 
E.g <Gamepad>/button*

Is the wider case handled elsewhere?

Reply:
Your right, now that I look at the documentation I doubt that this is sufficient. I just saw that the StringMatches() Method in InputControlPath.cs does exactly that (Line 534).
But it is only used in some Tests if I don't miss something. Maybe it is worth to refactor that for this use? @lyndon-unity

Reply:
see my new commit - I also saw there are cases where there can be "**" signs to indicate a wildcard. I think this case is not covered yet, it is also not part of the documentaiton, but maybe to keep it in mind it makes sense to create another bug for that?